### PR TITLE
New interaction icons

### LINKED ISSUE #154: Create new interact icons
image

### PR DESCRIPTION
**Pull request by floogulinc**
Closes #154

| Flower | Nest |
|---|---|
|  |  |
| localhost_4200_play_FSdSm102Ksyudd9YqXBE(Pixel 2) | localhost_4200_play_FSdSm102Ksyudd9YqXBE(Pixel 2) (1) |

### CODE REVIEW

File: src/app/components/play-round/play-round.component.ts [hunk 1]
```diff
@@ -49,7 +51,15 @@ export class PlayRoundComponent implements OnInit {
     map(([flowerMarkers, nestMarker]) => flowerMarkers.concat([nestMarker])),
   );
 
-  constructor(public studentRoundService: StudentRoundService, private sessionService: StudentSessionService) { }
+  constructor(
+    public studentRoundService: StudentRoundService,
+    private sessionService: StudentSessionService,
+    iconRegistry: MatIconRegistry,
+    sanitizer: DomSanitizer
+    ) {
```
Comment (helloworld12321) [inline src/app/components/play-round/play-round.component.ts:59]:
```suggestion
  ) {
```

Comment (helloworld12321) [reply src/app/components/play-round/play-round.component.ts]: Just a little thing :)

**Review by helloworld12321 — APPROVED** [review]
Looks wonderful—thank you so much, Paul!

**Review by kklamberty — APPROVED** [review]
Those look great! Thanks! I think that the flower one *might* look like the arrow is a stem and leaves, so we will want to watch out for that possibility and consider a different arrow shape that is more definitively arrow if needed. I think the context clues will mean it's not a problem.

**Comment by helloworld12321** [comment]
@kklamberty Oh, I see what you're saying :)
I don't think I'm worried with the ambiguity there—after all, if people look at the button and think "flower!", they're still more or less getting the right message! :stuck_out_tongue:

**Pull request merged**